LSP-metals.sublime-settings

```diff
@@ -33,11 +33,11 @@
   "settings": {
     "metals": {
       // When enabled, each method and variable that can have inferred types has them displayed.
-      "show-inferred-type": false,
+      "showInferredType": false,
```

Comment:
Why the change? Both are actually correct.

Reply:
Ah I didn't know ! 
My setup wasn't working and at some point I thought it was due to the key names :man_facepalming: 

I think they should be renamed anyway to match the metals doc which is camelcased
https://scalameta.org/metals/docs/integrations/new-editor#should-display-type-annotations-for-inferred-types

Reply:
We can rename it for sure, I was mostly wondering if something was broken 😅

Reply:
Nothing is broken :D